src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -255,7 +253,7 @@ public Schema fetchTableSchema(String tableName, FailureCollector collector) {
     Schema schema = null;
     try {
       schema = fetchTableSchema(tableName);
-    } catch (OAuthProblemException | OAuthSystemException | RuntimeException e) {
+    } catch (Exception e) {
       LOG.error("Error in connection - {}", e.getMessage());
```

Comment:
When logging an exception, always log the whole exception instead of just the message. Also, the message should give more context

```
LOG.error("Failed to fetch schema on table {}", tableName, e);
```